tests/test_util.py

```diff
@@ -19,15 +21,17 @@ def test_generate_clks(self):
             self.assertEqual(len(clk[0]), 1024)
             self.assertEqual(clk[0].count(), clk[2])
 
-    def test_popcount_vector(self):
-        bas = [util.generate_bitarray(1024) for i in range(100)]
+# Generate bit arrays that are combinations of words 0, 1, 2^63, 2^64 - 1
+# of various lengths between 1 and 65 words.
+def test_popcount_vector():
```

Comment:
Probably a good idea to keep the tests part of a `TestCase` - I know `nose` finds functions that have test in the name but we are now running with py.test

Reply:
Will leave the parameterised tests as top-level functions for now as it is not possible to put functions marked as `@pytest.parametrized` into `unittest.TestCase` classes (see [notes at the end](https://docs.pytest.org/en/latest/unittest.html)).